<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>510</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>开始检测</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_type">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>260</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>请选择VCU类型</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CV01A_220_001D00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CV02A_220_001A00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CV03A_220_001A00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CV03B_220_001A00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CV03C_220_001A00</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>191</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_timer">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>检测计时：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>秒</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_state_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>171</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>25</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_state_img">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>112</width>
       <height>112</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
